src/certifier_algorithms.cc

```diff
@@ -49,15 +50,30 @@ const char * Enc_method_rsa_4096                  = "rsa-4096";
 const char * Enc_method_rsa_4096_private          = "rsa-4096-private";
 const char * Enc_method_rsa_4096_public           = "rsa-4096-public";
 const char * Enc_method_rsa_4096_sha384_pkcs_sign = "rsa-4096-sha384-pkcs-sign";
- 
+
+const char *Enc_public_key_algorithms[] =
```

Comment:
HI, @jlmucb - I need your help in defining the list of public-key algorithms in this array.

And the list of symmetric-key algorithms defined on L62.

I took an initial stab, but seems like from the full list of `Enc_method_*` algorithm names, we should be able to separate out public-key and symmetric-key algorithms into two buckets.

Can you put down the list(s), so I can continue code-dev on this PR to address issue #174?